.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,68 @@
+name: Run tests
+
+on:
+  push:
+  release:
+    types:
+      - published
+
+jobs:
+  php-tests:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        php:
+          - 7.3
```

Comment:
I think this misses `7.2`, at least according to the similar changes in `composer.json` => `"php": "^7.2"`